src/infrastructure/services/GraphqlApollo.ts

```diff
@@ -26,6 +26,7 @@ export class GraphQlApolloService implements GraphQlService {
     return this.retryIfNeeded(() =>
       this.client
         .query<Response, Variables>({
+          fetchPolicy: 'no-cache',
```

Comment:
This is required so that when we re-fetch the spots (to get the comments), we do not use the cache. This implementation isn't great : all resources will bypass the cache.

We could :

1) pass the fetch policy as an option
2) use `cache.writeQuery` to manually add the new comment to the cache.

Reply:
I think we can merge it as it is for now. I am working a an implementation for 2. (will be opening PR soon)